src/nvim/diff.c

```diff
@@ -36,23 +42,45 @@
 #include "nvim/os/os.h"
 #include "nvim/os/shell.h"
 
-static int diff_busy = FALSE;    // ex_diffgetput() is busy
+static int diff_busy = kFalse;    // ex_diffgetput() is busy
```

Comment:
`kFalse` is of type `TriState`. The linter just wants this to be `false`. We wouldn't use a `TriState` for `int`.

Reply:
Thanks!